### PR TITLE
adds audio/comments icon to annotations

### PR DESCRIPTION
**Pull request by TS-Stefan**
also fixes annotation delete buttons hide issue.

### CODE REVIEW

**Comment by TS-Daniel16** [comment]
@TS-Stefan is this ready for review or are you working on it?

**Comment by TS-Stefan** [comment]
I am done

File: apps/koala-frontend/src/app/features/sessions/components/annotation/annotation.component.html [hunk 1]
```diff
@@ -2,16 +2,16 @@
   <svg [id]="'d3-labels-' + d3ContainerID" style="width: 30px; margin-top: 0px"></svg>
   <svg (window:resize)="onResize()" [id]="'d3-container-' + d3ContainerID" style="width: 100%"></svg>
   <svg
-    *ngIf="!displayMode && enableAnnotationDelete"
+    [ngStyle]="{ visibility: !displayMode && enableAnnotationDelete ? 'visibile' : 'hidden' }"
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/components/annotation/annotation.component.html:5]: there is a type in visible (visibile)

File: apps/koala-frontend/src/app/features/sessions/components/annotation/annotation.component.html [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
     [id]="'d3-delete-' + d3ContainerID"
     style="width: 44px; margin-left: 20px"
   >
     <foreignObject xmlns="http://www.w3.org/2000/svg" x="0" y="0" width="44" height="100%">
       <div *ngFor="let marker of markers">
         <button
-          *ngIf="marker.visible"
           pButton
           style="margin-bottom: 12px; margin-top: 12px"
+          [ngStyle]="{ visibility: marker.visible ? 'visibile' : 'hidden' }"
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/components/annotation/annotation.component.html:14]: Same as in line 5

**Review by TS-Daniel16 — CHANGES_REQUESTED** [review]

**Review by TS-Tino16 — APPROVED** [review]
LGTM

**Review by TS-Daniel16 — APPROVED** [review]

**Pull request merged**